tools/unix/generate_symbols.sh

```diff
@@ -18,7 +18,11 @@ fi
 export PYTHONDONTWRITEBYTECODE=1
 
 OMIM_PATH="${OMIM_PATH:-$(cd "$(dirname "$0")/../.."; pwd)}"
-OUT_PATH="$OMIM_PATH/out/release"
+
+if [ -z "${OUT_PATH}" ]; then
+  OUT_PATH="$OMIM_PATH/out/release"
```

Comment:
It is inconvenient to override env var. What is wrong with the hard-coded out/release path? Maybe it can be changed to `build/skin_generator_tool` instead?

Reply:
Because built once (with god knows what version of QT etc) it is always used and nobody knows that it may be outdated. I'd like to use always up-to-date tool from my regular output build folder.

Reply:
Then the fix should be different. E.g. check for the existing binary/create binary in `$OMIM_PATH/../omim-build-release`, `$OMIM_PATH/build`, `$OMIM_PATH/cmake-build-release`